### PR TITLE
python3: add few missing depends

### PR DESCRIPTION
**Pull request by dkwo**
@ahesford Forgot to explicitly add a couple of depends.

### CODE REVIEW

File: srcpkgs/python3-jupyter_console/template [hunk 1]
```diff
@@ -5,7 +5,8 @@ revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-ipython python3-ipython_ipykernel python3-jupyter_client
- python3-prompt_toolkit python3-Pygments python3-traitlets"
+ python3-prompt_toolkit python3-Pygments python3-traitlets
+ python3-platformdirs"
```
Comment (ahesford) [inline srcpkgs/python3-jupyter_console/template:9]: This package will need a revbump to see the updated dependency.

**Review by ahesford — CHANGES_REQUESTED** [review]
In addition to the modifications noted below, please rearrange the commits so that `python3-notebook_shim` appears in the history before the `jupyterlab` commit that adds the dependency.

File: srcpkgs/python3-jupyter_console/template (hunk 1 above)
Comment (dkwo) [reply srcpkgs/python3-jupyter_console/template:9]: Done.

File: srcpkgs/jupyter-notebook-shim/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'jupyter-notebook-shim'
+pkgname=jupyter-notebook-shim
```
[the remark this answers is not in the document]
Comment (dkwo) [reply srcpkgs/jupyter-notebook-shim/template]: Done.

**Comment by dkwo** [comment]
@tornaria @ahesford I believe this is the right way to fix it.

File: srcpkgs/python3-notebook_shim/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+build_style=python3-pep517
+make_install_target="dist/notebook_shim-${version}-*-*-*.whl"
+hostmakedepends="hatchling"
+depends="python3-jupyter_server"
+checkdepends="python3-pytest python3-jupyter_server python3-platformdirs"
+short_desc="Shim layer for notebook traits and config"
+maintainer="dkwo <npiazza@disroot.org>"
+license="BSD-3-Clause"
+homepage="https://github.com/jupyter/notebook_shim"
+distfiles="https://github.com/jupyter/notebook_shim/releases/download/v${version}/notebook_shim-${version}.tar.gz"
```
Comment (ahesford) [inline srcpkgs/python3-notebook_shim/template:14]: We prefer `${PYPI_SITE}/n/notebook_shim/notebook_shim-${version}.tar.gz` if it exists.

File: srcpkgs/python3-notebook_shim/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-notebook_shim'
+pkgname=python3-notebook_shim
+version=0.2.2
+revision=1
+build_style=python3-pep517
+make_install_target="dist/notebook_shim-${version}-*-*-*.whl"
```
Comment (ahesford) [inline srcpkgs/python3-notebook_shim/template:6]: Won't this be found automatically?

File: srcpkgs/python3-notebook_shim/template (hunk 3 above)
Comment (dkwo) [reply srcpkgs/python3-notebook_shim/template]: ok

File: srcpkgs/python3-notebook_shim/template (hunk 4 above)
Comment (dkwo) [reply srcpkgs/python3-notebook_shim/template]: yes

**Pull request closed without merging**